omero/users/history.txt

```diff
@@ -1,155 +1,72 @@
-updating code before the full public release of 5.3.0. Use at your own risk**.
+A full, production-ready release of OMERO 5.3.0; featuring a major re-working
+of OMERO.web and web apps; dropping support for Windows for the server;
+and introducing new user features and many fixes and performance improvements:
 
-Changes include:
+-  improved support for many file formats via Bio-Formats 5.3.4
+-  introduced ROI Folders
+-  new grid view layout for OMERO.web UI when displaying Screen Plate Well
+   data
+-  support for lookup tables and reverse intensity rendering
+-  color mapping for multiple channels without set colors has been improved to
+   use RGBRGB rather than RGBBB (i.e. to loop through red, green, blue rather
+   than setting all later channels to blue)
+-  support for histograms in the clients and server
+-  ability to filter by ratings in OMERO.web
+-  added 'Open With...' functionality to OMERO.web
+-  images can now be projected along T
+-  color of shapes is now handled according to the data model, using RGBA
+   rather than ARGB format
+-  improved performance for moving and deleting data
+-  Wells can now be annotated and searched by annotations
+-  enabled downloading/export of plate data
+-  improved reading of tables data
+-  script improvements including ability to create tiled images from big ROIs,
+   fixes for creating standard images from ROIs, and to stop the
+   Combine_Images script from ignoring pixel sizes set on the target images
+-  names for plates and images set in the metadata read by Bio-Formats are now
+   imported into OMERO and the filename (which was previously used) is only
+   used where an alternative has not been set
+-  many bug fixes
+
+Sysadmin changes include:
 
--  database patches to add support for new administration roles
--  removed restriction on name length
--  added support for enumeration changes
--  added new Grid View to both UI clients
--  introduced the new Web API
--  reactivated codomain support in the rendering Engine
 -  added support for Ice 3.6.3
--  fixed graph issues when performing chown or moving between groups
--  made web applications installable from PyPI
--  added option "Open With" feature to OMERO.web
-
-
-5.3.0-m4 (September 2016)
--------------------------
-
-Developer preview release - **only intended as a developer preview for
-updating code before the full public release of 5.3.0. Use at your own risk**.
-
-Changes include:
-
--  completed Web decoupling
--  improved CLI plugins
--  deprecated Apache
-
-5.3.0-m3 (August 2016)
-----------------------
+-  OMERO.web apps are now all installable via pip
+-  OMERO.web has been decoupled from the server and can now be deployed
+   separately
+-  dropped support for Windows for OMERO.server
+-  OMERO.web deployment via Apache is no longer supported
+-  CLI improvements
+-  options added for customizing the tree in OMERO.web
+-  changed DB setup script to require PostgreSQL 9.3
```

Comment:
I'm thinking I'll just remove that then, I'll add the update to version requirements on the sysadmin page